frontend/homepages/README.md

```diff
@@ -0,0 +1,8 @@
+# Project Homepages
+
+This directory contains homepages for kinetic and the research team on OpenStax.org.
+
+To develop the research page run `yarn run page:research:dev` and edit the research-homepage.tsx in the research directory.
+
```

Comment:
does this also apply for the homepage?

if so should we add:

```suggestion
To develop the homepage page run `yarn run page:homepage:dev` and edit the homepage.tsx in the homepage directory.
```

(unsure of the filename) lol

if this isnt integrated for homepage yet too just ignore this comment 😆